### PR TITLE
WNP 67.0.5 (#7128)

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
firefox/67.0.5/whatsnew/

## Issue / Bugzilla link
#7128

## Testing
**Demo** 
https://www-demo2.allizom.org/en-US/firefox/67.0.5/whatsnew/

- In about:config, set `identity.fxaccounts.autoconfig` preference to `https://stable.dev.lcip.org/`
- set `browser.uitour.testingOrigins` to `https://www-demo2.allizom.org/`.
- Create a new boolean preference (right click) and name it `browser.uitour.requireSecure`. Set its value to “false”.

**Locally**
add `FXA_ENDPOINT=https://latest.dev.lcip.org/` to .env

### CODE REVIEW

**Comment by amychurchwell** [comment]
**5/17:** Posting rough draft of page to allow interested parties to comment/collaborate. Design is still very much a work in progress. Not ready for final review.

File: media/js/base/mozilla-monitor-button.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    window.Mozilla = {};
+}
+
+Mozilla.MonitorButton = (function(Mozilla) {
+    'use strict';
+
+    var monitorButton = document.getElementById('fxa-monitor-submit');
+    var buttonURL = monitorButton.getAttribute('href');
+
+    // fetch request
```
Comment (amychurchwell) [inline media/js/base/mozilla-monitor-button.js]: @alexgibson howdy! -would love your thoughts on this fetch request, any gaps?

Comment (alexgibson) [reply media/js/base/mozilla-monitor-button.js]: @amychurchwell tested this locally and it seems to be doing the right thing to me 👍 

I'd suggest putting this up on demo so the FxA / Monitor folks can help vet the full flow?

**Comment by dzingeek** [comment]
**Final design mockups with final copy.**

**Logged in to an account**
WNP-67 5-logged-in 1312

**Not logged in to an account**
WNP-67 5-logged-out 1312

File: media/js/base/mozilla-monitor-button.js (hunk 1 above)
Comment (amychurchwell) [reply media/js/base/mozilla-monitor-button.js]: @groovecoder hi Luke! I threw up the current WIP page on demo so we can test the Monitor functionality. https://www-demo2.allizom.org/en-US/firefox/67.0.5/whatsnew/

It seems to be working locally, but I could not seem to get  the [fxa-dev-launcher](https://github.com/vladikoff/fxa-dev-launcher#basic-usage-example-in-os-x) to work for me. I tried `FXA_ENV=stage npm start`

Comment (groovecoder) [reply media/js/base/mozilla-monitor-button.js]: Sorry I missed this until now. Is this still going out as a WNP for 67.0.5? I got the 67 update and was sent here: https://www.mozilla.org/en-US/firefox/67.0/whatsnew/?oldversion=66.0.5

Comment (groovecoder) [reply media/js/base/mozilla-monitor-button.js]: And https://www-demo2.allizom.org/en-US/firefox/67.0.5/whatsnew/ doesn't seem to have anything more than a Monitor link?

Comment (amychurchwell) [reply media/js/base/mozilla-monitor-button.js]: @groovecoder This will be going out for 67.0.5, so the 67 update is expected behavior. 👍 
Do you have time to do a zoom call today? Should be more than a Monitor link, but I have been not able to replicate the behavior on local on demo and I think it's because my setup is not correct? (fxa-dev-launcher)

**Comment by amychurchwell** [comment]
@stephaniehobson hi Stephanie! This is in a good place for a code review. Eric mentioned Craig & I should ping you for trailhead reviews.

File: media/css/firefox/whatsnew/whatsnew-67.0.5.scss [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/hero';
+@import '../../protocol/components/state-fxa';
+
+body {
+    background: url('/media/img/firefox/whatsnew_67/noodles.png') no-repeat;
```
Comment (stephaniehobson) [inline media/css/firefox/whatsnew/whatsnew-67.0.5.scss]:
```suggestion
    background: $color-white url('/media/img/firefox/whatsnew_67/noodles.png') no-repeat;
```
TL;DR: add background colour declaration please.

Okay, reeeeeeeealy obscure edge case to consider here. When declaring the background shorthand if you don't set a background-colour it will set the colour to transparent. Back in the day browsers used to have a default background colour of grey. So this could have resulted in a grey background. I bring this up now because of the recent appearance of dark themes. I don't actually know if this will cause problems but I think it might be wise to be explicit here.

File: media/css/firefox/whatsnew/whatsnew-67.0.5.scss [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+}
+
+//* -------------------------------------------------------------------------- */
+// CTA
+
+.card {
+    background: #fff;
+    border-radius: 3px;
+    box-shadow: 1px 1px 10px 1px rgba(0, 0, 0, 0.21);
+    padding: $spacing-xl $spacing-2xl $spacing-xl $spacing-2xl;
```
Comment (stephaniehobson) [inline media/css/firefox/whatsnew/whatsnew-67.0.5.scss]: nit
```suggestion
    padding: $spacing-xl $spacing-2xl;
```

File: media/css/firefox/whatsnew/whatsnew-67.0.5.scss [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+
+    h2 {
+        @include text-display-sm;
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// Benefits
+
+.benefits {
```
Comment (stephaniehobson) [inline media/css/firefox/whatsnew/whatsnew-67.0.5.scss]: It would be nice to use the `c-` prefix for this component, but it's not a blocker.

File: media/css/firefox/whatsnew/whatsnew-67.0.5.scss [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+    }
+
+    .fxa-email-form-intro {
+        display: none;
+    }
+
+    .fxa-email-field-container {
+        .field {
+            label {
+                @include visually-hidden;
```
Comment (stephaniehobson) [inline media/css/firefox/whatsnew/whatsnew-67.0.5.scss]: 🔥

File: media/img/firefox/whatsnew_67/WNP-PDP-icon.svg [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="46" height="37" viewBox="0 0 46 37">
```
Comment (stephaniehobson) [inline media/img/firefox/whatsnew_67/WNP-PDP-icon.svg]: SVG FTW

File: media/js/base/mozilla-monitor-button.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        return resp.json();
+    }).then(function(r) {
+        // add retrieved flowBeginTime and flowId values to cta url
+        buttonURL += '&flowBeginTime=' + r.flowBeginTime;
+        buttonURL += '&flowId=' + r.flowId;
+        monitorButton.setAttribute('href', buttonURL);
+    }).catch(function() {
+        // silently fail, leaving flow_id and flow_begin_time as default empty value
+    });
+})(window.Mozilla);
```
Comment (stephaniehobson) [inline media/js/base/mozilla-monitor-button.js]: Since we're not using the Mozilla object inside the function we don't need to pass it in.

```suggestion
})();
```

File: media/js/base/mozilla-monitor-button.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+    var buttonURL = monitorButton.getAttribute('href');
+
+    // fetch request
+    var supportsFetch = 'fetch' in window;
+
+    if (!supportsFetch) {
+        return;
+    }
+
+    var destURL = monitorButton.getAttribute('action') + 'metrics-flow';
```
Comment (stephaniehobson) [inline media/js/base/mozilla-monitor-button.js]: I don't think action is a valid attribute for `a` and it does get used on `form` so I'm worried about confusing the browser. Maybe you want `data-action` instead?

File: media/static-bundles.json [hunk 9]
```diff
@@ -1019,6 +1025,17 @@
...
+    {
+      "files": [
+        "js/base/uitour-lib.js",
+        "js/base/mozilla-fxa.js",
+        "js/base/mozilla-fxa-init.js",
+        "js/base/mozilla-fxa-form.js",
+        "js/base/mozilla-monitor-button.js",
+        "protocol/js/protocol-modal.js"
+      ],
+      "name": "firefox_whatsnew_trailhead"
```
Comment (stephaniehobson) [inline media/static-bundles.json]: Nit: I like to name the CSS and JS bundles the same thing unless we think they're going to be used separately.

File: media/css/firefox/whatsnew/whatsnew-67.0.5.scss (hunk 2 above)
Comment (amychurchwell) [reply media/css/firefox/whatsnew/whatsnew-67.0.5.scss]: 👏 super interesting! will do.

File: bedrock/base/templates/macros.html [hunk 10]
```diff
@@ -328,7 +328,7 @@ <h3>{{ _('Your download link was sent.') }}</h3>
 {%- endmacro %}
 
 {% macro monitor_button(utm_source='', utm_campaign='', button_text='', button_class='mzp-c-button mzp-t-product') -%}
-  <a data-action="{{ settings.FXA_ENDPOINT }}" href="https://fx-breach-alerts.herokuapp.com/oauth/init?utm_source={{ utm_source }}&utm_campaign={{ utm_campaign }}&form_type=email&entrypoint=mozilla.org-whatsnew6705" class="{{ button_class }}" id="fxa-monitor-submit" data-button-name="Continue">
+  <a data-action="{{ settings.FXA_ENDPOINT }}" href="https://monitor.firefox.com/oauth/init?utm_source={{ utm_source }}&utm_campaign={{ utm_campaign }}&form_type=email&entrypoint=mozilla.org-whatsnew6705" class="{{ button_class }}" id="fxa-monitor-submit" data-button-name="Continue">
```
Comment (stephaniehobson) [inline bedrock/base/templates/macros.html]: Good catch 😅

**Comment by stephaniehobson** [comment]
R+ 🖥
